plex-service/src/main/java/com/cleverpine/plex/repository/future/EpisodesRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.cleverpine.plex.repository.future;
+
+import com.cleverpine.plex.entity.future.EpisodeEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
```

Comment:
It's good practice to annotate the repository class with the annotation @Repository.